cmd/golangci-lint/main.go

```diff
@@ -3,20 +3,43 @@ package main
 import (
 	"fmt"
 	"os"
+	"runtime/debug"
 
 	"github.com/golangci/golangci-lint/pkg/commands"
 	"github.com/golangci/golangci-lint/pkg/exitcodes"
 )
 
 var (
+	goVersion = "unknown"
+
 	// Populated by goreleaser during build
 	version = "master"
 	commit  = "?"
 	date    = ""
 )
 
+//nolint:gochecknoinits
+func init() {
+	if info, available := debug.ReadBuildInfo(); available {
+		goVersion = info.GoVersion
+
+		if date == "" {
+			version = info.Main.Version
+			commit = fmt.Sprintf("(unknown, mod sum: %q)", info.Main.Sum)
+			date = "(unknown)"
+		}
+	}
+}
```

Comment:
I see this is just moved but why is it in an `init` function and not at the start of `main` since this is in the main package?

Reply:
:thinking: I don't know :smile: 

I will change that.